source/agora/consensus/data/Transaction.d

```diff
@@ -76,7 +76,31 @@ public struct Transaction
             hashPart(output, dg);
     }
 
-
+    /// Transaction type hashing for unittest
+    unittest
```

Comment:
Please move the unittest to module scope, outside the Transaction class.

Reply:
Move to outside completed